app/scripts/modules/core/src/pipeline/config/stages/managed/PublishDeliveryConfigExecutionDetails.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { get } from 'lodash';
+
+import { ExecutionDetailsSection, IExecutionDetailsSectionProps, StageFailureMessage } from 'core/pipeline';
+import { IGitTrigger } from 'core/domain';
+import { SETTINGS } from '../../../../config';
+
+export function PublishDeliveryConfigExecutionDetails(props: IExecutionDetailsSectionProps) {
+  const { stage } = props;
+  const trigger = props.execution.trigger as IGitTrigger;
+  const errorDetailsAvailable = stage.isFailed && !stage.failureMessage && get(stage.context, 'error') != null;
+  const manifestDirectory =
+    SETTINGS.managedDelivery.manifestBasePath + (stage.context.directory ? stage.context.directory : '');
```

Comment:
```suggestion
    SETTINGS.managedDelivery?.manifestBasePath + (stage.context.directory ?? '');
```
`managedDelivery` is optional, so we want to guard against it being null; `??` is the nullish coalescing operator - basically a shorthand version of what you're doing there already.

Reply:
Got it! I guessed the "elvis" operator (`?:`, like in Kotlin) which didn't work but was lazy to read the docs to see if there was an equivalent. Thanks for letting me know!